app/codes/state_updater.py

```diff
@@ -181,8 +181,8 @@ def add_block_reward(cur, creator, blockindex):
     if creator is None:
         return False
     reward = 0
-    RATIO = 2 / 3
-    STARTING_REWARD = 1000
+    RATIO = 0.99
+    STARTING_REWARD = 100
     block_step = math.ceil(float(blockindex) / 1000000)
     reward = STARTING_REWARD * pow(RATIO, (block_step - 1)) * pow(10, NEWRL_TOKEN_DECIMAL)
```

Comment:
this should be int() or math.floor()

Reply:
you mean 
`reward = math.floor(STARTING_REWARD * pow(RATIO, (block_step - 1)) * pow(10, NEWRL_TOKEN_DECIMAL))` ?